### ISSUE TITLE
PULSE: Self-Supervised Photo Upsampling via Latent Space Exploration of Generative Models

### ISSUE THREAD

**Issue by AkiraTOSEI** [AkiraTOSEI/ML_papers#101]
# TL;DR
In super-resolution, instead of taking loss that making  high-resolution image from low-resolution image, they are looking for a latent space that would result in the same image when reverting to a lower resolution after a getting higher resolution from the pre-trained model. A method that does not require learning a large network and is not tied to network choices.
![image](https://user-images.githubusercontent.com/24524018/92562160-750a5a00-f2b0-11ea-9c34-c9eb866160ff.png)

# Why it matters:

# Paper URL
https://arxiv.org/abs/2003.03808

# Submission Dates(yyyy/mm/dd)

# Authors and institutions

# Methods

# Results

# Comments
